src/lib/settings/schema/types/User.js

```diff
@@ -24,6 +24,14 @@ class UserType extends SchemaType {
 		throw language.get('RESOLVER_INVALID_USER', piece.key);
 	}
 
+	serialize(data) {
```

Comment:
Shouldn't this be on the base class as what's default?
`return data.id || data.name || ...`